portfolio/src/main/java/com/google/sps/servlets/UserComment.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
```

Comment:
Value classes like this should generally implement `equals`, `hashCode` and `toString` methods, so that it can be consistently used in collections and other places.

You might find AutoValue in Google's Guava library useful, so that you don't need to manually write all of these: see https://github.com/google/auto/blob/master/value/userguide/index.md for the introduction.

Reply:
This is cool :)